src/sass/components/_loading.scss

```diff
@@ -0,0 +1,37 @@
+.loading-indicator {
+  background-color: rgba($white, 0.7);
+  border: 2px solid rgba($primary, 0.7);
+  bottom: 20px;
+  left: 50%;
+  padding: 10px;
+  position: fixed;
+  transform: translateX(-50%);
+  width: calc(100% - 20px);
+  z-index: 9999;
+
+  .loading-bar {
+    animation: loading-effect 3s infinite linear;
+    background-color: yellow;
+    background-size: 50px 50px;
+    background-image: linear-gradient( -45deg, lighten($primary, 15%) 25%, $white 25%, $white 50%, lighten($primary, 15%) 50%, lighten($primary, 15%) 75%, $white 75%,$white );
+    border: 1px solid lighten($primary, 15%);
+    border-radius: 50px;
+    height: 20px;
+  }
+}
+
+@keyframes loading-effect {
```

Comment:
Gewoon een tip, moet je niet perse aanpassen als je het bullshit vind ;) Ik plaats mijn keyframes altijd allemaal bovenaan de file, omdat die zo iets losstaand is, ook niet in media queries komt